docs/source/release-notes.rst

```diff
@@ -2,6 +2,13 @@ Release Notes
 =============
 
 
+Version 0.8.3
+_____________
+
+- BREAKING CHANGES:
+  - class `Json` from `hopeit.dataobjects.jsonify` renamed to `Payload` and move to `hopeit.dataobjects.payload` for more intuitive usage of @dataobject decorated object. Deprecation warning starting version 0.8.2 till 0.9.0
```

Comment:
"It will be deprecated in a future version" (no need to say which version). 

Correct deprecated since version according to changes in version.py